controllers/configurationpolicy_controller.go

```diff
@@ -158,20 +160,47 @@ func (r *ConfigurationPolicyReconciler) PeriodicallyExecConfigPolicies(freq uint
 			skipLoop = true
 		}
 
+		scheduledPolicyCount := 0
+
+		// This is done every loop cycle since the channel needs to be variable in size to account for the number of
+		// policies changing.
+		policyQueue := make(chan *policyv1.ConfigurationPolicy, len(policiesList.Items))
+		doneQueue := make(chan struct{}, len(policiesList.Items))
+
+		for i := 0; i < int(r.EvaluationConcurrency); i++ {
```

Comment:
I think we need to validate that `r.EvaluationConcurrency` is not 0, otherwise bad things could happen here. A user could think setting it to `0` would maximize the concurrency.